src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -16,6 +17,25 @@ public partial class PackageDependencyView : UserControl
 
         private WorkspaceModel currentWorkspace;
 
+        private ViewLoadedParams loadedParams;
+        private PackageDependencyViewExtension dependencyViewExtension;
```

Comment:
This is the funny part I did not feel comfortable doing when I was talking about this with @mjkkirschner yesterday

Reply:
Yes, I wasn't sure too about storing the object here. There are not a lot of properties that are being exposed with this object.